dayrepo/snippets/templates/snippet_checklist.html

```diff
@@ -0,0 +1,13 @@
+{% block page_title %}投稿{% endblock %}
+{% block title %}投稿{% endblock %}
+{% load static %}
+{% block content %}
+<form method="post">
+        <h1>点検項目項目<h2>※出発前に車両</h2></h1>
```

Comment:
見出しの中に見出しは基本的にないかな
表記を点検項目項より小さくしたいという意味なのであれば、後々CSSに記載するべきではあるが、以下でどうだろう。

```suggestion
        <h2 style="display: inline;">点検項目項目</h2>※出発前に車両
```


FYI
https://tsuneaka.club/?p=586

Reply:
妙にひねったこと書いた私が悪いけど、デザインは基本的に基本動作が整ってからにしたいです。
また、デザインの変更は確定事項であり、複数想定されるので、[デザイン対応のissue](https://github.com/bob-g12/dayrepo-for-transportation/issues/40)を個別で作成しておきます。

Reply:
@bob-g12 
どちらかというとHTMLタグの使い方に関する指摘なので、であればこうしてください
```suggestion
        <h2>点検項目項目※出発前に車両</h2>
```